crates/roc_std/src/roc_dict.rs

```diff
@@ -5,6 +5,16 @@ use core::{
     mem::{align_of, ManuallyDrop},
 };
 
+/// At the moment, Roc's Dict is just an association list. It's lookups O(n) but
+/// we haven't grown such big programs that it's a problem yet!
+///
+/// We do some things in this data structure that only make sense because the
+/// memory is managed in Roc:
+///
+/// 1. We don't implement an `IntoIterator` that iterates over owned values,
```

Comment:
you can do the following so rustdoc will generate automatic links:

```suggestion
/// 1. We don't implement an [`IntoIterator`] that iterates over owned values,
```